bcipy/signal/tests/model/test_keyinput.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+from bcipy.signal.model.keyinput import compute_keyinput_probs
+
+
+class TestKeyinput(unittest.TestCase):
+    def setUp(self):
+        self.inquiry = ["A", "E", "I", "O", "U"]
+        self.symbol_set = self.inquiry + ["Y"]
+        self.error_prob = 0.05
+
+    def test_user_likes_inquiry(self):
+        proceed = True
+        results = compute_keyinput_probs(self.inquiry, self.symbol_set, self.error_prob, proceed)["KEYINPUT"]
+
+        p_shown = (1 - self.error_prob) / len(self.inquiry)
+        p_not_shown = (self.error_prob) / (len(self.symbol_set) - len(self.inquiry))
+        expected = [p_shown] * 5 + [p_not_shown]
```

Comment:
Given the small symbol set, rather than computing the expected values it might be more clear to just manually list these out.